cluster/gce/gci/configure-helper.sh

```diff
@@ -335,7 +342,8 @@ function ensure-local-ssds() {
   local i=0
   for ssd in /dev/disk/by-id/google-local-ssd-*; do
     if [ -e "${ssd}" ]; then
-      local devicenum=`echo ${ssd} | sed -e 's/\/dev\/disk\/by-id\/google-local-ssd-\([0-9]*\)/\1/'`
+      local devicenum
+      devicenum=$(echo "${ssd}" | sed -e 's/\/dev\/disk\/by-id\/google-local-ssd-\([0-9]*\)/\1/')
```

Comment:
since we're touching this, let's make it more readable
```suggestion
      devicenum=$(echo "${ssd}" | sed -e 's@/dev/disk/by-id/google-local-ssd-\([0-9]*\)@\1@')
```

Reply:
Very good suggestion. I was not sure how popular deliminators other than / were here, so I didn't dare to change that. Update coming up!